src/playlist-maker.js

```diff
@@ -24,13 +34,20 @@ var playlistMaker = function(player, plist) {
     if (plist && isArray(plist)) {
       list = plist.slice();
       player.playlist.currentItem(0);
+
+      window.setTimeout(function() {
```

Comment:
I thought that given that we are actually setting the source and tracks and all that of the first item of the playlist that we should trigger this asynchronously so the browser has a chance to react to those changes before someone does something with the updated playlist.